src/org/camunda/latera/bss/connectors/PlanadoV2.groovy

```diff
@@ -214,6 +214,24 @@ class PlanadoV2 {
     sendRequest('post', 'jobs', body: payload).getOrNull()
   }
 
+  Map updateJob(def jobId, Map data) {
+    logger.info('Updating job')
+
+    LinkedHashMap payload = [:].with {
+      if (data.containsKey('description')) {
+        description = data.description
+      }
+
+      it
+    }
+
+    if (payload.size() == 0) {
+      throw new Exception("No params for update")
+    }
+
+    sendRequest('patch', "jobs/${jobId}", body: payload).getOrNull()
```

Comment:
Have you test it? There was a bug with PATCH method support in underlying http library: http-builder-ng/http-builder-ng#118

Reply:
We have nothing to test it